pages/1.11/installing/upgrading/index.md

```diff
@@ -78,46 +79,56 @@ These steps must be performed for version upgrades and cluster configuration cha
+- Verify that all your masters are in a healthy state: 
+   - Check the Exhibitor UI to confirm that all masters have joined the quorum successfully (the status indicator will show green). The Exhibitor UI is available at `http://<dcos_master>:8181/`.
+   - Verify that `curl http://<dcos_master_private_ip>:5050/metrics/snapshot` has the metric `registrar/log/recovered` with a value of `1` for each master.
+
 
 ## Bootstrap Node
 
+[enterprise]
+### Enterprise users 
+[/enterprise]
+
 Choose your desired security mode and then follow the applicable upgrade instructions.
 
 - [Installing DC/OS 1.11 without changing security mode](#current-security)
 - [Installing DC/OS 1.11 in permissive mode](#permissive)
 - [Installing DC/OS 1.11 in strict mode](#strict)
 
-# <a name="current-security"></a>Installing DC/OS 1.11 without changing security mode
+#### Installing DC/OS 1.11 without changing security mode 
 This procedure upgrades a DC/OS 1.10 cluster to DC/OS 1.11 without changing the cluster's [security mode](/1.11/installing/ent/custom/configuration/configuration-parameters/#security-enterprise).
 
 1.  Copy your existing `config.yaml` and `ip-detect` files to an empty `genconf` folder on your bootstrap node. The folder should be in the same directory as the installer.
-1.  Merge the old `config.yaml` into the new `config.yaml` format. In most cases the differences will be minimal.
+2.  Merge the old `config.yaml` into the new `config.yaml` format. In most cases the differences will be minimal.
 
     **Important:**
 
     *  You cannot change the `exhibitor_zk_backend` setting during an upgrade.
     *  The syntax of the `config.yaml` may be different from the earlier version. For a detailed description of the current `config.yaml` syntax and parameters, see the [documentation](/1.11/installing/ent/custom/configuration/configuration-parameters/).
-1. After updating the format of the config.yaml, compare the old config.yaml and new config.yaml. Verify that there are no differences in pathways or configurations. Changing these while upgrading can lead to catastrophic cluster failures.
-1.  Modify the `ip-detect` file as desired.
-1.  Build your installer package.
+3. After updating the format of the config.yaml, compare the old config.yaml and new config.yaml. Verify that there are no differences in pathways or configurations. Changing these while upgrading can lead to catastrophic cluster failures.
+4.  Modify the `ip-detect` file as desired.
+5.  Build your installer package.
 
     1.  Download the `dcos_generate_config.ee.sh` file.
-    1.  Generate the installation files. Replace `<installed_cluster_version>` in the below command with the DC/OS version currently running on the cluster you intend to upgrade, for example `1.8.8`.
+    2.  Generate the installation files. Replace `<installed_cluster_version>` in the below command with the DC/OS version currently running on the cluster you intend to upgrade, for example `1.8.8`.
```

Comment:
Markdown supports auto numbering and doesn't require the user to explicitly make a change to these numbers.